api/control.proto

```diff
@@ -175,7 +175,11 @@ message ListNodesRequest {
 	message Filters {
 		repeated string names = 1;
 		repeated string id_prefixes = 2;
+    // Labels refers to engine labels, which are labels set by the user on the
```

Comment:
Nitpick:  Tab/whitespace seems off from the existing code.

Reply:
i always screw that up